src/classes/Foo5.cls

```diff
@@ -0,0 +1,15 @@
+public class Foo5 {
+    void foo() {
+        
+        
+        
+        String myString = 'MY_HARDCODED_STRING';
+        Blob hardCodedIV = Blob.valueOf('Hardcoded IV 123');
```

Comment:
Constant `hardCodedIV` does not match naming conventions. Please rename (hint: `myVariable`).